data-client/src/main/java/org/wikipedia/login/LoginClient.java

```diff
@@ -33,7 +33,13 @@
 public class LoginClient {
     @Nullable private Call<MwQueryResponse> tokenCall;
     @Nullable private Call<LoginResponse> loginCall;
-
+    /**
+     * userLanguage
+     * It holds the value of the user's device language code.
+     * For example, if user's device language is English it will hold En
+     * The value will be fetched when the user clicks Login Button in the LoginActivity
+     */
+    @NonNull private String userLanguage;
```

Comment:
Sorry, another very small thing: Could you please add a new line after this line? Thanks!

Reply:
Done!!